tests/webtests/CreateConnectionRequestTest.php

```diff
@@ -61,15 +61,17 @@ public function test_a_connection_request_can_be_created()
         self::assertSelectorIsDisabled('#connection_request_container_send');
         self::assertSelectorIsEnabled('#connection_request_container_cancel');
 
+        self::assertSelectorTextContains('.empty-connection-list-message', 'No items have been added yet. Add items to this list using the + button, then click the Send button.');
```

Comment:
Suggestion: inject the translator in this test, and simply check if `__('entity.create_connection_request.emptyConnectionListPlaceholder')` is on the page. That way, when the translation changes, you do not have to update you test.